.release-notes/3650.md

```diff
@@ -0,0 +1,17 @@
+## Graceful error handling on `consume` expressions
+
+The fix ensures 
+* `consume (consume variable).field` results in a graceful compilation error
+* `consume f().field` results in similar compilation error as `consume f()`
+* Allow `(consume obj).field` expressions
```

Comment:
this doesn't fit with the rest 

"this ensures allow blah" doesn't read well. can you update this bullet to align with "this ensures"